app/services/devices.js

```diff
@@ -416,6 +448,49 @@ DevicesService.prototype.forget = function forget(device, done) {
   });
 };
 
+DevicesService.prototype.create = function create(device, done) {
+  readConfigFile(function onReadConfigFile(readConfigFileErr, deviceConfig) {
+    if (readConfigFileErr) {
+      done(readConfigFileErr);
+      return;
+    }
+
+    deviceConfig = {
```

Comment:
What do you think about moving the code responsible for creating/mounting this config to another function? sth. like `getDeviceConfig()`.

Reply:
We are reading the existing file but doing nothing with its configuration file, right? It seems you are rewriting it completely, so should we still read it? Maybe it'd be important when having an `edit` feature but I'm not sure for now.

Reply:
I totally agree with you @netoax. Actually, after doing a little investigation inside the code that I implemented, I saw that the `readConfigFile` was going to leave a bug for us to resolve. So, anyway, I got rid of the `readConfigFile`.